apis/iam/v1beta1/diff_test.go

```diff
@@ -0,0 +1,856 @@
+					{
+						Role:    "roles/viewer",
+						Members: []Member{"user:viewer@example.com", "user:other@example.com"},
+					},
+				},
+			},
+			actual: &IAMPolicySpec{
+				Bindings: []IAMPolicyBinding{
+					{
+						Role:    "roles/viewer",
+						Members: []Member{"user:other@example.com", "user:viewer@example.com"},
+					},
+				},
+			},
+			want: &structuredreporting.Diff{},
+		},
+		{
+			name: "binding condition differs",
+			desired: &IAMPolicySpec{
+				Bindings: []IAMPolicyBinding{
+					{
+						Role:    "roles/viewer",
+						Members: []Member{"user:viewer@example.com"},
+						Condition: &IAMCondition{
+							Title:      "title",
+							Expression: "expression",
+						},
+					},
+				},
+			},
+			actual: &IAMPolicySpec{
+				Bindings: []IAMPolicyBinding{
+					{
+						Role:    "roles/viewer",
+						Members: []Member{"user:viewer@example.com"},
+						Condition: &IAMCondition{
+							Title:      "other-title",
+							Expression: "expression",
+						},
+					},
+				},
+			},
+			want: &structuredreporting.Diff{
+				Fields: []structuredreporting.DiffField{
+					{
+						ID: "spec.bindings[role=roles/viewer]",
+						Old: IAMPolicyBinding{
+							Role:    "roles/viewer",
+							Members: []Member{"user:viewer@example.com"},
+							Condition: &IAMCondition{
+								Title:      "title",
+								Expression: "expression",
+							},
+						},
+						New: IAMPolicyBinding{
+							Role:    "roles/viewer",
+							Members: []Member{"user:viewer@example.com"},
+							Condition: &IAMCondition{
+								Title:      "other-title",
+								Expression: "expression",
+							},
+						},
+					},
+				},
+			}},
+		{
+			name: "empty desired bindings",
+			desired: &IAMPolicySpec{
+				Bindings: []IAMPolicyBinding{},
+			},
+			actual: &IAMPolicySpec{
+				Bindings: []IAMPolicyBinding{
+					{
+						Role:    "roles/viewer",
+						Members: []Member{"user:viewer@example.com"},
+					},
+				},
+			},
+			want: &structuredreporting.Diff{
+				Fields: []structuredreporting.DiffField{
+					{
+						ID:  "spec.bindings[role=roles/viewer]",
+						Old: "absent in desired spec",
+						New: "present in actual spec",
+					},
+				},
+			},
+		},
+		{
+			name: "empty actual bindings",
+			desired: &IAMPolicySpec{
+				Bindings: []IAMPolicyBinding{
+					{
+						Role:    "roles/viewer",
+						Members: []Member{"user:viewer@example.com"},
+					},
+				},
+			},
+			actual: &IAMPolicySpec{
+				Bindings: []IAMPolicyBinding{},
+			},
+			want: &structuredreporting.Diff{
+				Fields: []structuredreporting.DiffField{
+					{
+						ID:  "spec.bindings[role=roles/viewer]",
+						Old: "present in desired spec",
+						New: "absent in actual spec",
+					},
+				},
+			},
+		},
+		{
+			name: "audit config added",
+			desired: &IAMPolicySpec{
+				AuditConfigs: []IAMPolicyAuditConfig{
+					{
+						Service: "allServices",
+						AuditLogConfigs: []AuditLogConfig{
+							{
+								LogType: "DATA_READ",
+							},
```

Comment:
The test `binding condition differs` covers a modification to an existing condition. Please add test cases to cover the addition and removal of a `Condition` from a binding to improve coverage.